src/Sulu/Bundle/AdminBundle/Resources/js/containers/SingleSelection/SingleSelection.js

```diff
@@ -45,7 +45,9 @@ class SingleSelection extends React.Component<Props> {
 
         this.singleSelectionStore = new SingleSelectionStore(resourceKey, value, locale, detailOptions);
         this.changeDisposer = reaction(
-            () => (this.singleSelectionStore.item ? this.singleSelectionStore.item.id : undefined),
+            () => !this.singleSelectionStore.item
```

Comment:
this change doesnt do anything right? 🤔

Reply:
okay i understand. not sure if it a good idea to implicitly rely on the difference between `undefined` and `null`. i guess this will lead to confusion in the future 🙈